figures/scripts/figS7-seqcenter-barplot.R

```diff
@@ -0,0 +1,86 @@
+# S Spielman for CCDL, 2023
+# Create panel A for Figure S7 that shows how histologies (cancer groups)
+# are not balanced across RNA library preparations (polyA vs. stranded)
+
+
+#### Libraries -----------------------------------------------------------------
+
+library(tidyverse)
+library(ggpubr)
+
+#### Directories and files -----------------------------------------------------
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+figures_dir <- file.path(root_dir, "figures")
+output_dir <- file.path(figures_dir, "pdfs", "supp", "figS7", "panels")
+
+output_pdf <- file.path(output_dir, "RNA_library_barplot.pdf")
+
+metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
+palette_file <- file.path(figures_dir, "palettes", "broad_histology_cancer_group_palette.tsv")
+binary_pal_file <- file.path(figures_dir, "palettes", "binary_color_palette.tsv")
+
+##### Read in and prepare data for plot ----------------------
+
+metadata_df <- read_tsv(metadata_file, guess_max = 10000)
+palette_df <- read_tsv(palette_file)
+binary_pal_df <- read_tsv(binary_pal_file)
+
+# Get the RNA samples and combine with `cancer_group_display` information
+library_cancer_group_df <- metadata_df %>%
+  filter(experimental_strategy == "RNA-Seq") %>%
+  # We want one row per patient here
+  mutate(dup_participant = duplicated(Kids_First_Participant_ID)) %>%
+  filter(!dup_participant) %>%
+  # Keep RNA_library and cancer group for each
```

Comment:
Hi @sjspielman just catching up on this. I think there may be a bug here, which may be due to using participants instead of samples here - can you check out `PT_9S6WMQ92` - this patient has two different events sequenced, one is stranded and one is polyA (medullo). Seems like we capture stranded in the plot but not the polyA. Caught my eye.